app/routes/docs.contributing.configuration/route.mdx

```diff
@@ -179,5 +179,16 @@ account for GCN production by registering the email address
       </Description>
       <Default>Kafka producer is disabled</Default>
     </tr>
+    <tr>
+      <Key>`ARC_TABLES_PORT`, `ARC_DB_EXTERNAL`</Key>
+      <Description>
+        Variables for local running DynamoDB. Set the port to `ARC_TABLES_PORT`
+        to 8000 and `ARC_DB_EXTERNAL` to `true` to use the local DynamoDB docker
+        image and streams plugins
+      </Description>
+      <Default>
+        Default DynamoDB behavior from Architect and streams are not enabled
+      </Default>
+    </tr>
```

Comment:
Would you please create a follow-up issue in this repository and assign it to yourself, to note that we need to automate this new configuration?